kernel/kernel-api/src/main/java/io/delta/kernel/internal/TransactionImpl.java

```diff
@@ -84,9 +84,8 @@ public class TransactionImpl implements Transaction {
   private final Protocol protocol;
   private final SnapshotImpl readSnapshot;
   private final Optional<SetTransaction> setTxnOpt;
-  private final Optional<List<Column>> clusteringColumnsOpt;
+  private final Optional<List<Column>> newClusteringColumnsOpt;
```

Comment:
Can we again here add javadocs `/** */` to this variable indicating what the different states mean?